### PR TITLE
Cocoapods support

### PR DESCRIPTION
**Pull request by AFcgi**
Now there are a support cocoapods as well. If you want to publish a new version, please note the following steps:

1. `make cocoapods` and commit the new binary of iblinter
2. update the version in IBLinter.podspec
3. `release.sh` with the new version

### CODE REVIEW

File: IBLinter.podspec [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Pod::Spec.new do |s|
+  s.name           = 'IBLinter'
+  s.version        = '0.2.0'
+  s.summary        = 'A linter tool for Interface Builder.'
+  s.homepage       = 'https://github.com/kateinoigakukun/IBLinter'
+  s.license        = { :type => 'MIT', :file => 'LICENSE' }
+  s.author         = { 'Yuta Saito' => 'kateinoigakukun@gmail.com' }
+  s.source         = { :git => "https://github.com/kateinoigakukun/IBLinter.git", :branch => "feature/cocoapods" }
```
Comment (kateinoigakukun) [inline IBLinter.podspec]: Should remove `:branch => "feature/cocoapods"`?

File: README.md [hunk 2]
```diff
@@ -43,6 +53,11 @@ else
 fi
 ```
 
+Alternatively, if you've installed IBLinter via CocoaPods the script should look like this:
+
+```bash
```
Comment (kateinoigakukun) [inline README.md]: Please replace `bash` with `sh`.
I'm sorry for being so fussy.

**Comment by AFcgi** [comment]
@kateinoigakukun I have implemented your comments.

**Comment by kateinoigakukun** [comment]
@AFcgi thanks!

**Comment by AFcgi** [comment]
@kateinoigakukun Please release a new version of IBLinter and bump up to 0.2.0. After that you run `pod spec lint`and `pod trunk push IBLinter.podspec`. Only after you've done all that, IBLinter can be installed via Cocoapods.

**Pull request merged**